packages/relay/src/lib/eth.ts

```diff
@@ -1878,57 +1813,32 @@ export class EthImpl implements Eth {
     return EthImpl.numberTo0x(block.count);
   }
 
-  private getTransactionFromContractResults(contractResults: any, requestIdPrefix?: string) {
-    if (!contractResults || contractResults.length == 0) {
-      // contract result not found
+  private static formatContractResult(cr: any) {
```

Comment:
This would have been good to move to the formatter class along with test scenarios